quit/web/modules/endpoint.py

```diff
@@ -148,7 +154,7 @@ def provenance():
 
     if q:
         try:
-            query_type = parse_query_type(q)
+            queryType, parsedQuery = parse_query_type(q)
             graph = quit.store.store
         except UnSupportedQueryType as e:
```

Comment:
Please add a test, when `UnSupportedQueryType` is raised also for the other occurences of Exceptions.